katello/rubygem-hammer_cli_foreman_virt_who_configure/rubygem-hammer_cli_foreman_virt_who_configure.spec

```diff
@@ -0,0 +1,79 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_foreman_virt_who_configure
+%global confdir hammer
+
+Summary: Hammer CLI commands for configuring Virt Who for Katello
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.3
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/theforeman/hammer-cli-foreman-virt-who-configure
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 1.0.1-7
```

Comment:
This also obsoletes a much newer version than the SCL-version.

Reply:
Straight up removed since this was a copy/paste fail.